ads/genieessp.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {validateDataExists, writeScript} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function genieessp(global, data) {
+  validateDataExists(data, ['vid', 'zid']);
+
+  global.data = data;
+  writeScript(global, 'https://js.gsspcln.jp/l/amp.js');
```

Comment:
Thank you for your reviewing.
We can't use `loadScript` because our ad tags require to load scripts synchronously.
In the future version of the ad tags, we'd like to support asynchronous loading.